pkg/test/context.go

```diff
@@ -48,23 +47,8 @@ type CleanupOptions struct {
 type cleanupFn func() error
 
 func (f *Framework) newTestCtx(t *testing.T) *TestCtx {
-	var prefix string
-	if t != nil {
-		// TestCtx is used among others for namespace names where '/' is forbidden
-		prefix = strings.TrimPrefix(
-			strings.Replace(
-				strings.ToLower(t.Name()),
-				"/",
-				"-",
-				-1,
-			),
-			"test",
-		)
-	} else {
-		prefix = "main"
-	}
-
-	id := prefix + "-" + strconv.FormatInt(time.Now().Unix(), 10)
+	// TestCtx is used among others for namespace names where '/' is forbidden and must be 63 characters or less
+	id := "osdk-e2e-" + uuid.New()
```

Comment:
I like this approach much better than my previous code change. This weekend I'd still like to add unit tests around NewTestCtx, but please let me know thoughts on this approach.

Reply:
It is ok for me. @joelanford have you any objection?